solarwinds_apm/configurator.py

```diff
@@ -46,7 +47,6 @@
 )
 from solarwinds_apm.apm_fwkv_manager import SolarWindsFrameworkKvManager
 from solarwinds_apm.apm_meter_manager import SolarWindsMeterManager
-from solarwinds_apm.apm_noop import Reporter
```

Comment:
This was actually the wrong import used for type checking before. Removed this and added `solarwinds_apm.extension.oboe.Reporter` below.